mitmproxy/contentviews/graphql.py

```diff
@@ -30,7 +30,12 @@ def is_graphql_query(data):
 
 
 def is_graphql_batch_query(data):
-    return isinstance(data, list) and isinstance(data[0], dict) and "query" in data[0]
+    return (
+        len(data) > 0
```

Comment:
You've put this in a different order than what I suggested. This will fail for anything that doesn't have a length, e.g. `null`

```
HTTP/2.0 200
content-type: application/json; charset=utf-8
content-length: 4

null
```

Reply:
Yeah missed that case. I changed it back to your order.